File: Heat/ViewModels/Invoices/InvoiceCreateViewModel.vb
Imports Heat.Models
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels

    ''' <summary>
    ''' Modello 
    ''' </summary>
    ''' <remarks></remarks>
    Public Class InvoiceCreateViewModel
        <Key> _
        Property ID As Integer

        <Display(name:="Cliente")> _
        Property CustomerName As String

        <Display(name:="Numero documento")> _
        Property TempNumber As String

        <Display(name:="Data emissione")> _
        Property EmissionDate As Date


    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Invoices/InvoiceCreateViewModel.cs
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Invoices
{

    /// <summary>
    /// Modello 
    /// </summary>
    /// <remarks></remarks>
    public class InvoiceCreateViewModel
	{
		[Key()]
		public int ID { get; set; }

		[Display(Name = "Cliente")]
		public string CustomerName { get; set; }

		[Display(Name = "Numero documento")]
		public string TempNumber { get; set; }

		[Display(Name = "Data emissione")]
		public System.DateTime EmissionDate { get; set; }


	}

}

